server/graphql-schema.js

```diff
@@ -762,15 +766,15 @@ const graphqlSchema = gql`
         Indicates the type of issue. 'CHANGES_REQUESTED' or 'FEEDBACK'.
         'FEEDBACK' is the default type.
         """
-        feedbackType: IssueFeedbackType!
```

Comment:
Is this supposed to be IssueFeedbackType or String?

Reply:
Yep, this is supposed to be `IssueFeedbackType`. It points to the enum defined on L747 which may have gotten ignored during some previous merge?